fableous-be/handlers/session.go

```diff
@@ -25,9 +25,10 @@ func (m *module) SessionGetAllByClassroomID(classroomID string) (sessionInfos []
 			ID:          session.ID,
 			Title:       session.Title,
 			Description: session.Description,
-			Pages:       session.Pages,
-			Completed:   session.Completed,
-			CreatedAt:   session.CreatedAt,
+			// TODO
```

Comment:
is todo here for adding names of student?

Reply:
Fixed